cdc/kv/client.go

```diff
@@ -270,7 +273,26 @@ func NewCDCClient(ctx context.Context, pd pd.Client, kvStorage tikv.Storage, cre
 		}{
 			conns: make(map[string]*connArray),
 		},
+		currentVersionCache: tidbkv.Version{Ver: 0},
 	}
+
+	go (func() {
+		ticker := time.NewTicker(tsoQueryInterval)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-ticker.C:
```

Comment:
if there is no region to resolve lock, we still query the tso from pd every two seconds?